docs/user-guide/researcher/model-testing-during-federated-training.md

```diff
@@ -149,8 +153,25 @@ exp.set_set_test_metric(MetricTypes.ACCURACY)
 exp.run(rounds=2, increase=True)
 ```
 
+## Using Validation facility with heavy datasets
 
+In some specific cases, you may have some very huge datain your datasets, such as 3D images, or you want a huge validation set, that you can load on `Node`s only through batches, in order to avoid `Nodes` to crash due to lack of Memory. `Fed-BioMed` provides an option to compute validation metrics through batches, in the `TrainingArgument`: `test_batch_size`.
 
+By setting `test_batch_size`, selected metrics will be computed over batches instead of computing metric using the whole dataset (which is the default behaviour). For now, `Fed-BioMed` doesnot provide a way to reconcile all these metrics under one metric (such as computing the mean of Accuracy for instance). If you want to do so, please consider defining your own `Validation step` by write your own `testing_step` method in the `TrainingPlan` (see below for further datails).
```

Comment:
```suggestion
Validation will be computed over batches instead of computing metric using the whole dataset (which is the default behaviour) if `test_batch_size` is set. For now, `Fed-BioMed` does not provide a way to reconcile all these metrics under one metric (such as computing the mean of Accuracy for instance). If you want to do so, please consider defining your own `Validation step` by writing your own `testing_step` method in the `TrainingPlan` (see below for further details).
```